app/models/hub_edos/student.rb

```diff
@@ -0,0 +1,32 @@
+module HubEdos
+  class Student
+    def self.get_student(uid)
+      student_hash = HubEdos::V2::Student.new(user_id: uid).get
+      student_feed = student_hash[:feed]
+      {
+        termsInAttendance: self.get_max_terms_in_attendance(student_feed),
+        academicLevel: self.get_student_academic_level(student_feed),
+      }
+    end
+
+    def self.get_max_terms_in_attendance(student_feed)
+      if statuses = student_feed['academicStatuses']
+        return statuses.collect {|s| s['termsInAttendance']}.sort.last
+      end
+    end
+
+    def self.get_student_academic_level(student_feed)
+      current_registration = get_current_registration(student_feed)
+      type_code = current_registration['academicCareer']['code'] == 'LAW' ? 'EOT' : 'BOT'
+      level = current_registration['academicLevels'].find {|al| al['type']['code'] == type_code }
+      level['level']['description']
+    end
+
+    def self.get_current_registration(student_feed)
+      current_term_id = Berkeley::Terms.fetch.current.campus_solutions_id
+      if registrations = student_feed['registrations']
+        return registrations.find { |r| r['term']['id'] == current_term_id }
+      end
+    end
+  end
+end
```

Comment:
I think it might make sense to refactor this to a class that take the UID in an initializer and #max_terms_in_attendance and #academic_level are methods on that object. What do you think?

Reply:
@developish - Done